src/main/app/fees/feesClient.ts

```diff
@@ -34,49 +35,48 @@ export class FeesClient {
    * @returns {Promise.<number>} promise containing the fee amount in pounds
    */
   static async calculateHearingFee (claimValue: number): Promise<number> {
-    const fee: CalculationOutcome = await this.calculateFee(hearingFeeCode, claimValue)
-    return MoneyConverter.convertPenniesToPounds(fee.amount)
+    return this.calculateFee(hearingFeeEvent, claimValue, defaultChannel)
+      .then((outcome: FeeOutcome) => (outcome.amount))
   }
-
   /**
-   * Calculates the fee based on fee code and amount
+   * Calculates the fee based on fee event and amount
```

Comment:
Please update JSDoc to mention that channel is also being taken into consideration.